include/wil/windowing.h

```diff
@@ -14,6 +14,8 @@
 #include <WinUser.h>
 #include <exception>
 
+#include "common.h"

```

Comment:
This was missing before & is needed for the `WIL_HAS_EXCEPTIONS` checks

Reply:
I'm surprised that wasn't a build break.  Does this OSS project build WIL with the compiler warnings cranked up to the maximum and /analyze on the command line?

Reply:
It would just `ifdef` to false. I'm assuming that the tests include a different WIL header before this one, so it wasn't caught.